docs/best-practices.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Best Practices'
+---
+
+Emotion is an extremely flexible library, but this can make it intimidating, especially for new users. This guide contains several recommendations for how to use Emotion in your application. Keep in mind, these are only recommendations, not requirements!
+
+## Recommendations
+
+### Use TypeScript and object styles
+
+You don't get Intellisense or type checking when using CSS strings, e.g. `` css`color: blue` ``. You can improve the developer experience and prevent style bugs by using TypeScript and object styles, which enable Intellisense and _some_ static type checking:
+
+```tsx
+const myCss = css({
+  color: 'blue',
+  grid: 1 // Error: Type 'number' is not assignable to type 'Grid | Grid[] | undefined'
+})
+```
+
+### Share styles via component reuse
```

Comment:
This section is a little bit opinionated. When I've started to look into it I was somewhat expecting to see a comparison of how both approaches compose styles etc but that was not the case.

Reply:
Hopefully it is a bit less opinionated now.